### PR TITLE
Add Kuro manga Reader for Android

### PR DESCRIPTION
**Pull request by WadieBenabdouh**
Added **Kuro Reader Pro** app for reading Manga as well as Comics for Android devices.

1. Cracked APK.
2. Supports all files types.

### CODE REVIEW

File: Literature/Manga.md [hunk 1]
```diff
@@ -93,5 +93,7 @@ Remember to always use an AdBlocker like [uBlock Origin](https://github.com/gorh
 	- [**Neko**](https://tachiyomi.org/forks/Neko/) - This is a MangaDex specific fork of Tachiyomi and TachiyomiJ2K. Contains features specific to Mangadex that those versions cannot offer except for other sources, local sources, and editting manga.  
 	- [**TachiyomiJ2K**](https://tachiyomi.org/forks/TachiyomiJ2K/) - This fork takes a new approach to the design of the base app along with several other enhancements.
 
+- [**Kuro Reader Pro (Cracked APK) (<span style="color:red">Android</span>)**](https://dlandroid.com/kuro-reader-pro-donate-cbz-cbr-cbt-cb7-reader-apk/) - Create your library and have quick access to all of your books. You can add as many folders you like and have all of your manga and comics organized alphabetically by year and author series.
```
Comment (Helmasko) [inline Literature/Manga.md:96]: We only allow official download links, not cracked or 3rd party

Comment (WadieBenabdouh) [reply Literature/Manga.md:96]: > We only allow official download links, not cracked or 3rd party

Yeah sorry I thought that would be useful, and yeah there's also the free copy of this app available on PlayStore.

**Comment by WadieBenabdouh** [comment]
Looking forward to contributing more...

**Comment by WadieBenabdouh** [comment]
Pull Request opened I'll be working on it a few minutes.

File: Literature/Manga.md (hunk 1 above)
Comment (WadieBenabdouh) [reply Literature/Manga.md:96]: @Helmasko I'll be working on a new PR, I deleted my recent fork.

**Pull request closed without merging**